tr55/tables.py

```diff
@@ -404,10 +404,10 @@
     'urban_grass':          {'nlcd': 21, 'ki': 0.7, 'cn': {'a': 68, 'b': 79, 'c': 86, 'd': 89}},  # noqa
     'woody_wetland':        {'nlcd': 90, 'ki': 1, 'cn': {'a': 98, 'b': 98, 'c': 98, 'd': 98}},  # noqa
     'herbaceous_wetland':   {'nlcd': 95, 'ki': 1, 'cn': {'a': 98, 'b': 98, 'c': 98, 'd': 98}},  # noqa
-    'chaparral':            {'nlcd': 95, 'ki': 1, 'cn': {'a': 98, 'b': 98, 'c': 98, 'd': 98}},  # noqa
-    'tall_grass_prairie':   {'nlcd': 95, 'ki': 1, 'cn': {'a': 98, 'b': 98, 'c': 98, 'd': 98}},  # noqa
-    'short_grass_prairie':  {'nlcd': 95, 'ki': 1, 'cn': {'a': 98, 'b': 98, 'c': 98, 'd': 98}},  # noqa
-    'desert':               {'nlcd': 95, 'ki': 1, 'cn': {'a': 98, 'b': 98, 'c': 98, 'd': 98}},  # noqa
+    'chaparral':            {'nlcd': 52, 'ki': 1, 'cn': {'a': 35, 'b': 56, 'c': 70, 'd': 77}},  # noqa
+    'tall_grass_prairie':   {'nlcd': 85, 'ki': 0.6, 'cn': {'a': 35, 'b': 56, 'c': 70, 'd': 77}},  # noqa XXX
```

Comment:
Yeah, I could not find find solid numbers for type 85.  I used the ki from short grass and the curve numbers from "brush in fair condition".  I also used similarly imputed event mean concentrations.